src/lucky_migrator/runner.cr

```diff
@@ -54,6 +54,12 @@ class LuckyMigrator::Runner
     end
   end
 
+  def ensure_migrated!
+    if pending_migrations.size > 0
```

Comment:
Very minor comment, but you could do `if pending_migrations.any?`

Reply:
Ah that's what it is. I wasn't sure what the correct method was, thanks.